packages/@aws-cdk/aws-lambda-python/README.md

```diff
@@ -83,14 +111,18 @@ You may create a python-based lambda layer with `PythonLayerVersion`. If `Python
 or `Pipfile` or `poetry.lock` with the associated `pyproject.toml` at the entry path, then `PythonLayerVersion` will include the dependencies inline with your code in the
 layer.
 
-```ts
-new lambda.PythonFunction(this, 'MyFunction', {
-  entry: '/path/to/my/function',
-  runtime: Runtime.PYTHON_3_6,
-  layers: [
-    new lambda.PythonLayerVersion(this, 'MyLayer', {
-      entry: '/path/to/my/layer', // point this to your library's directory
-    }),
-  ],
+## Custom Bundling
+
+Custom bundling can be performed by using custom Docker images for bundling dependencies. If using a custom Docker image for bundling, the dependencies are installed with `pip`, `pipenv` or `poetry` by using the `Packaging` class.
+
+A different bundling Docker image that is in the same directory as the function can be specified as:
+
+ ```ts
+const entry = '/path/to/function';
+const image = DockerImage.fromBuild(entry);
```

Comment:
It would be cool to address some of the open issues in this example. I know there are a couple of open issues that ask about providing custom build args that we could show.

Reply:
Added an example for using Code Artifact below, which was also what brought me to contributing this. And added documentation for customizing other build args.